scripts/linters/find_banned_deps.rb

```diff
@@ -0,0 +1,17 @@
+# frozen_string_literal: true
```

Comment:
Could we make it more generic? E.g., have an array
```ruby
whitelist = {
 ["time"] => [],
 ["fvm", "fvm3"] => ["forest_shim"]
}
```
And then traverse the collection without any custom logic? Or something along those lines?

Reply:
Yepp, good suggestion!